packages/core/src/baseclient.ts

```diff
@@ -155,7 +156,10 @@ export abstract class BaseClient<B extends Backend, O extends Options> implement
   ): Promise<SentryResponse> {
     return this.buffer.add(
       (async () => {
-        const event = await this.getBackend().eventFromMessage(message, level, hint);
+        const event = isPrimitive(message)
+          ? await this.getBackend().eventFromMessage(`${message}`, level, hint)
```

Comment:
The other way around, I suppose?

Reply:
Also, should we instead use `serialize` here?

Reply:
No, this way. If it's a primitive, then capture it as a message, if complex value (obj/arr) then fallback to `eventFromException`. It will also handle serialization so we don't have to do it here.

Reply:
Aaah, gotcha